packages/uilib/src/lib/plugins/type-designer/editor-events/event-types.ts

```diff
@@ -0,0 +1,20 @@
+import { BayType, IEDType, LDeviceType, SubstationType, VoltageLevelType } from "../types/nodes";
```

Comment:
To add to my last comment : here what is the difference with the editor-events.ts file ? 🙃